File: 02-0314/04-Salary/Salary.c
#include <stdio.h>

int main() {
    int monthly_salary;
    int annual_income;
    
    printf("Enter your monthly salary: ");
    scanf("%d",&monthly_salary);
    
    annual_income = 12 * monthly_salary;
    
    printf("\n");
    printf("Your annual income is %d\n", annual_income);
    
    return 0;
}

File: 02-0314/04-Salary/Salary.java
import java.util.Scanner;

public class Salary {
    public static void main (String[] args) {
        int monthly_salary;
        int annual_income;
        Scanner input = new Scanner(System.in);

        System.out.print("Enter your monthly salary: ");
        monthly_salary = input.nextInt();
        
        annual_income = 12 * monthly_salary;
        
        System.out.println("");
        System.out.printf("Your annual income is %d\n", annual_income);
    }
}